### DISCUSSION TITLE
vs2022 Why vcpkg installed libmysql:x86-windows-static after Will the compilation link to Boost?

### DISCUSSION THREAD

**Discussion by xengine-qyt** [microsoft/vcpkg#42421] [Q&A] [answered]
Boost_thread-vc143-mt-x32-1_86.lib (thread. cpp. obj): error LNK2038: Checked "RuntimeLibrary" mismatches: the value "MT_StaticRelease" does not match the value "MD_DynamicRelease" (in dllmain. obj)

I found that if the vcpkg connection attribute in the project attribute is changed to static, it will be connected to the boost database. What is the reason?

it this bug?

**Comment by dg0yt** [comment, marked as answer]
Do you try to mix a dynamic-CRT project with a static-CRT windows triplet?
Maybe you want to use `x86-windows-static-md` (dynamic CRT)?

**Reply by xengine-qyt** [reply]
> Do you try to mix a dynamic-CRT project with a static-CRT windows triplet? Maybe you want to use `x86-windows-static-md` (dynamic CRT)?

all use x86-windows-static, and other open source libraries do not have this problem.just libmysql will cause this problem
